gnome-initial-setup/pages/account/gis-account-page.c

```diff
@@ -204,6 +204,21 @@ on_local_user_cached (GtkWidget      *page_local,
 }
 
 static void
+on_shared_user_created (GtkWidget    *page_local,
+                        ActUser        *user,
+                        char           *password,
+                        GisAccountPage *page)
+{
+    const gchar *language;
```

Comment:
Please check the indentation here too, it got switched to four-spaces instead of two-spaces by mistake. And the function parameters got misaligned.